### PR TITLE
Updates to fps docs.

### PR DESCRIPTION
**Pull request by joubert-macrometa**

### CODE REVIEW

File: docs/photoniq/fingerprinting/index.md [hunk 1]
```diff
@@ -21,28 +21,86 @@ When a new user visits your site, Digital Fingerprinting will:
...
+
+## Best Practices
+
+### 1st Party vs 3rd Party Cookies
+
+The Photoniq Fingerprint service uses [third-party cookies](first-third-party-cookie) by default, which may affect the accuracy of the identification process. To ensure high levels of identification accuracy, it is recommended to use a customer subdomain or a cloud proxy (such as Akamai). By employing this approach, the cookies set by the Fingerprint service will be treated as [first-party cookies](first-third-party-cookie), which will enhance the identification precision.
+
+### Fingerprint Browser Agent on Page Load
+
+The JavaScript agent is loaded at page load time, and the function to get the visitor ID is run right away. For more information, click [here]().
```
Comment (joubert-macrometa) [inline docs/photoniq/fingerprinting/index.md]: I need help to get this to point the the "page load" section of the getting started guide.

File: docs/photoniq/fingerprinting/index.md [hunk 2]
```diff
@@ -21,28 +21,86 @@ When a new user visits your site, Digital Fingerprinting will:
...
+
+### Visitor Traffic
+
+The visitor traffic report returns the number of good bots, bad bots, unknown bots, and humans who accessed your website. The report returns data in five-minute intervals over a date/time range of your choosing.
+
+## Best Practices
+
+### 1st Party vs 3rd Party Cookies
+
+The Photoniq Fingerprint service uses [third-party cookies](first-third-party-cookie) by default, which may affect the accuracy of the identification process. To ensure high levels of identification accuracy, it is recommended to use a customer subdomain or a cloud proxy (such as Akamai). By employing this approach, the cookies set by the Fingerprint service will be treated as [first-party cookies](first-third-party-cookie), which will enhance the identification precision.
```
Comment (joubert-macrometa) [inline docs/photoniq/fingerprinting/index.md]: Can you verify that these links are pointing to the new pages I created.

File: docs/photoniq/fingerprinting/index.md (hunk 1 above)
Comment (diana-macrometa) [reply docs/photoniq/fingerprinting/index.md]: You mean link to it? I will add a link.

Comment (joubert-macrometa) [reply docs/photoniq/fingerprinting/index.md]: Yes.  Link to it.  Thx.

**Comment by diana-macrometa** [comment]
@joubert-macrometa - I reorganized the content a bit, please take a look and let me know if you agree with the changes.

**Comment by joubert-macrometa** [comment]
Looks good to me

**Review by diana-macrometa — APPROVED** [review]

**Pull request merged**